### ISSUE TITLE
The input dimension in detector.

### ISSUE THREAD

**Issue by ChenhanXmu** [lfz/DSB2017#32]
In detector step, why the input dimension are different in training (1L,128L, 128L, 128L) )and testing (1L, 208L, 208L, 208L))? It's mean the input of the network is variable?

**Comment by lfz** [comment]
Yes, it is variable, but recently we found that the changing of shape may lead to unwanted changing of output. Please be careful about this
